### DISCUSSION TITLE
How to redirect to single-tenant Azure AD auth page for a static Jekyll site hosted on Static Web App?

### DISCUSSION THREAD

**Discussion by motoko89** [Azure/static-web-apps#705] [Q&A] [answered]
Hi all, I followed https://docs.microsoft.com/en-us/azure/static-web-apps/authentication-custom?tabs=aad and created the `staticwebapp.config.json` as below in the root folder:
``````
{
  "routes": [
    {
      "route": "/*",
      "allowedRoles": ["authenticated"]
    }
  ],
  "auth": {
    "identityProviders": {
      "azureActiveDirectory": {
        "registration": {
          "openIdIssuer": "https://login.microsoftonline.com/my-tenant-id/v2.0",
          "clientIdSettingName": "AZURE_CLIENT_ID",
          "clientSecretSettingName": "AZURE_CLIENT_SECRET"
        }
      }
    }
  }
}
``````

However, navigating to the site now just show:
``````
401: Unauthorized

You don’t have permissions for this page. Please check your login info and permissions and try again. Learn more
``````

How do I make the redirection to sign in page?

**Comment by miwebst** [comment, marked as answer]
Hi @motoko89, you probably will want to add an unauthenticated sign in route above the /*. After adding that you should be able to login.

However for automatic redirection to the sign-in page, you probably want to use a response override: https://docs.microsoft.com/en-us/azure/static-web-apps/configuration#response-overrides

**Comment by motoko89** [comment]
That works!
